package.json

```diff
@@ -280,7 +280,6 @@
 		"mkdirp": "^1.0.4",
 		"ncp": "^2.0.0",
 		"nock": "^12.0.3",
-		"node-sass": "^6.0.0",
 		"node-sass-package-importer": "^5.3.2",
```

Comment:
We keep the importer, since the same API is shared with dart sass. So we can still use it for the same purpose.